overrides/macos-override.json

```diff
@@ -510,6 +510,17 @@
                                 "reason": "Adwall appears which reappears when dismissed. The adwall prevents interaction with the page."
                             }
                         ]
+                    },
+                    "monetate.net": {
+                        "rules": [
+                            {
+                                "rule": "monetate.net",
+                                "domains": [
+                                    "kleen-ritecorp.com"
+                                ],
+                                "reason": "blank page, and product carousels not working - https://app.asana.com/0/1200277586140538/1205303551800514/f"
```

Comment:
Same as above, here's the link to the issue description: https://github.com/duckduckgo/privacy-configuration/issues/1231